astropy/modeling/tests/test_fitters.py

```diff
@@ -22,7 +22,12 @@
 from ...utils.data import get_pkg_data_filename
 from ...tests.helper import pytest
 from .utils import ignore_non_integer_warning
+from ...utils.exceptions import AstropyUserWarning
+from pkg_resources import EntryPoint
+from ..fitting import populate_entry_points
 from ...stats import sigma_clip
+import mock
```

Comment:
Yes, mock is definitely an optional dependency. Again, we could probably make the case for including it as a test requirement (since it's actually now part of the standard library in recent versions of Python 3). Actually, maybe it makes sense to have a ``try...execpt`` to try importing from the standard library first.